tasks/server-tests/server.test.mjs

```diff
@@ -220,31 +230,34 @@ describe.each([
     server.close()
   })
 
-  it("doesn't error out on unknown args", async () => {
-    child = execa.command(`${commandString} --foo --bar --baz`)
-    await new Promise((r) => setTimeout(r, TIMEOUT))
-
-    const res = await fetch('http://localhost:8910/about')
-    const body = await res.text()
-
-    expect(res.status).toEqual(200)
-    expect(body).toEqual(
-      fs.readFileSync(
-        path.join(__dirname, './fixtures/redwood-app/web/dist/about.html'),
-        'utf-8'
-      )
-    )
+  it('errors out on unknown args', async () => {
+    try {
+      await $`yarn node ${commandString} --foo --bar --baz`
+      expect(true).toEqual(false)
+    } catch (p) {
+      expect(p.exitCode).toEqual(1)
+      expect(p.stdout).toEqual('')
+      expect(p.stderr).toMatchSnapshot()
+    }
   })
+
+  it.todo('changing redwood.toml web.port works')
+  it.todo("if apiHost isn't set, and apiUrl isn't fully qualified, exits")
+  it.todo(
+    "if api host isn't set, and api url is fully qualified, hits upstream"
+  )
+  it.todo(
+    'if api host is set, and api url is fully qualified, also breaks cause it tries to proxy?'
+  )
```

Comment:
Please make these read better. "it if ..." sounds weird 😉 
The first one could be something like "it exits if apiH..."

Reply:
yeah these were just notes to myself really but happy to be a bit more coherent

Reply:
done in https://github.com/redwoodjs/redwood/pull/9809/commits/39194318513027c2803c35a916778eedab0db57f